### ISSUE TITLE
Subscription layout

### ISSUE THREAD

**Issue by Tsagu** [YePpHa/YouTubeCenter#2252]
I have my youtube center Dev version set to not do grid subscriptions, but instead list. Well now it only does grid, and when clicking list, it stays grid.

**Comment by Cephsus** [comment]
Yup same here, clicking list "flow=2" dosnt do anything. Maybe YT screwed something up, since i cant get it to work on other browsers either.

**Comment by tomtom9749** [comment]
Confirmed here as well. Youtube list view does nothing at all. Redirects to youtube gird.

**Comment by Slosten** [comment]
It's an issue with YouTube itself, not YouTube Center. Go check the YouTube support forums, there's already a bunch of threads about it. Hopefully it's a glitch and not intentional, although it wouldn't surprise me if this was an attempt by Google to kill plugins like this one.

**Comment by pingufetch5** [comment]
Same problem here on Firefox, for now we'll just have to turn off Grid subscriptions in Youtube Center.

**Comment by Leifman27** [comment]
yep same thing here, i'm sure YePpHa will find a solution when he sees it

**Comment by sangohan** [comment]
as a work around, just usethe mobile view till it gets fixed up: https://m.youtube.com/?#/feed/subscriptions

**Comment by Primeey** [comment]
[It's a youtube problem not youtube center](https://www.reddit.com/r/youtube/comments/4lytye/youtubes_subscriptions_page_wont_even_go_to_list/d3rhpk8)

**Comment by werian** [comment]
Funny thing, this thing as the OP stated only appeared after installing the youtube center, it was totally fine before i got YTC
